src/simplicial-hott/04-extension-types.rzk.md

```diff
@@ -265,6 +265,30 @@ We refer to another form as an "extension extensionality" axiom.
   := (ext-htpy-eq I ψ ϕ A a f g , extext I ψ ϕ A a f g)
 ```
 
+Sometimes, an even weaker form of extension extensionality suffices.
+
+```rzk title="Very weak extension extensionality"
+#def NaiveExtExt
+  : U
+  :=
+    ( I : CUBE) →
+    ( ψ : I → TOPE) →
+    ( ϕ : ψ → TOPE) →
+    ( A : ψ → U) →
+    ( a : (t : ϕ) → A t) →
+    ( f : (t : ψ) → A t [ϕ t ↦ a t]) →
+    ( g : (t : ψ) → A t [ϕ t ↦ a t]) →
+    ( (t : ψ) → (f t = g t) [ϕ t ↦ refl]) →
+    ( f = g)
+
+#def naiveweakext-extext
```

Comment:
Should this be `naiveextext-extext` or the type should be renamed to `NaiveWeakExt`?

```suggestion
#def naiveextext-extext
```

Reply:
Oh, that's a typo! Should be `naiveextext`..